CIME/case/case_submit.py

```diff
@@ -307,6 +307,10 @@ def check_case(self, skip_pnl=False, chksum=False):
         for comp in self.get_values("COMP_CLASSES"):
             if comp == "CPL":
                 continue
+            compname = self.get_value("COMP_{}".format(comp))
+            # ignore stub components in this test.
+            if compname.startswith("S"):
```

Comment:
I'm concerned this might be too general but not sure how else to identify stub models in case_submit.

Reply:
We have a convention that if a component name begins with 's' it is a stub component.
This is an assumption already used in many places in cime.

Reply:
Actually I could refine this a little: 
if compname == "s{comp}".format(comp)

I'll do that.